packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -80,33 +80,6 @@ export function labelRefresh(node: vscode.TreeItem): void {
     node.label = node.label.endsWith(" ") ? node.label.substring(0, node.label.length - 1) : node.label + " ";
 }
 
-/*************************************************************************************************************
```

Comment:
Was this removed because it is a duplicate of the `refreshTree` function in `ProfilesUtils.ts`?

Reply:
yes. That's the exact reason